detox/e2e/test/smoke_test/channels.e2e.js

```diff
@@ -0,0 +1,59 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {toChannelScreen} from '@support/ui/screen';
+
+import {Setup, Channel} from '@support/server_api';
+
+describe('Unread channels', () => {
+    let newChannel;
+    let aChannel;
+    let zChannel;
+
+    beforeAll(async () => {
+        const {user, channel, team} = await Setup.apiInit();
+        newChannel = channel;
+
+        let ch;
+
+        ch = await Channel.apiCreateChannel({type: 'O', prefix: 'a-channel', teamId: team.id});
+        aChannel = ch.channel;
+        await Channel.apiAddUserToChannel(user.id, aChannel.id);
+
+        ch = await Channel.apiCreateChannel({type: 'O', prefix: 'z-channel', teamId: team.id});
+        zChannel = ch.channel;
+        await Channel.apiAddUserToChannel(user.id, zChannel.id);
```

Comment:
nit: Looks weird to me. Got me thinking on whether we need to revisit on how the API calls are returning data. How about like below?
```suggestion
        ({channel: aChannel} = await Channel.apiCreateChannel({type: 'O', prefix: 'a-channel', teamId: team.id}));
        await Channel.apiAddUserToChannel(user.id, aChannel.id);

        ({channel: zChannel} = await Channel.apiCreateChannel({type: 'O', prefix: 'z-channel', teamId: team.id}));
        await Channel.apiAddUserToChannel(user.id, zChannel.id);
```